calicoctl/commands/datastore/migrate/migrateipam.go

```diff
@@ -120,6 +128,38 @@ func (m *migrateIPAM) PullFromDatastore() error {
 		if !ok {
 			return fmt.Errorf("Could not convert %+v to an AllocationBlock", item.Value)
 		}
+
+		// Update node names in the block to match the Kubernetes node
+		if m.nodeMap != nil {
```

Comment:
Is this ever going to actually be nil?

Reply:
It *shouldn't* ever be nil but I thought I should add the check just in case since it'll cause things to throw nil pointer exceptions.